components/Slider/src/MDCSlider.h

```diff
@@ -367,6 +368,17 @@ IB_DESIGNABLE
  */
 @property(nonatomic, assign) BOOL shouldEnableHapticsForAllDiscreteValues;
 
+/**
+ The font of the discrete value label.
+
+ This font will come into effect only when @c numberOfDiscreteValues is larger than 0 and when @c
+ shouldDisplayDiscreteValueLabel is
+ @c YES.
+
+ Defaults to [[MDCTypography fontLoader] regularFontOfSize:12].
+ */
+@property(nonatomic, strong, null_resettable) UIFont *discreteValueLabelFont;
```

Comment:
Please add unit tests for both the default value and the effect of setting `nil` (the reset effect).

Reply:
done